### PR TITLE
pretty/latex printing of S.Naturals0

### LINKED ISSUE #9848: Defining printing for `S.Naturals0`
```
>>> init_printing(use_unicode=True)
>>> S.Naturals
ℕ
>>> S.Naturals0
ℕ
```

There should should be some difference in their printing may be putting a `N0`(0 in subscript) for `S.Naturals0`.

### PR DESCRIPTION
**Pull request by leosartaj**
closes #9848

### CODE REVIEW

**Comment by leosartaj** [comment]
Ping @aktech @gxyd

**Comment by aktech** [comment]
Looks good. +1

File: sympy/printing/latex.py [hunk 1]
```diff
@@ -1552,6 +1552,9 @@ def _print_EmptySet(self, e):
     def _print_Naturals(self, n):
         return r"\mathbb{N}"
 
+    def _print_Naturals0(self, n):
+        return r"\mathbb{N_0}"
```
Comment (aktech) [inline sympy/printing/latex.py]: Btw, `\mathbb{W}` is used for whole numbers in LaTeX, but since in SymPy we use `N0`, so this should be OK.

**Comment by leosartaj** [comment]
Thanks @aktech

**Pull request merged**